Update tests for GNOME localsearch and remove call to feature removed in the tracker to tinysparql transition.

Source 1: https://gitlab.gnome.org/GNOME/nautilus/-/commit/0e6e3946ebfcbaf1c7e2ebbb1300dd3a016ee0e9.patch
Source 2: https://gitlab.gnome.org/GNOME/nautilus/-/commit/493858117e0e40052160b8e6df05bb4025e41d6c.patch
Source 3: https://gitlab.gnome.org/GNOME/nautilus/-/commit/493858117e0e40052160b8e6df05bb4025e41d6c.patch


From 0e6e3946ebfcbaf1c7e2ebbb1300dd3a016ee0e9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ant=C3=B3nio=20Fernandes?= <antoniof@gnome.org>
Date: Mon, 10 Jun 2024 22:41:33 +0100
Subject: [PATCH] general: React to tracker projects rename

Tracker SPARQL is now TinySPARQL. Tracker Miners is now LocalSearch.

The old DBus and library APIs still work so there is no rush there.

But the executables and git repositories need to be updated now.

https://discourse.gnome.org/t/renaming-tracker-git-and-tracker-sparql-git-and-changing-default-branch/21461
---
 README.md                                                | 2 +-
 build-aux/flatpak/org.gnome.Nautilus.json                | 8 ++++----
 data/tracker/meson.build                                 | 9 +++++----
 .../org.gnome.Nautilus.Tracker3.Miner.Extract.service.in | 4 ++--
 .../org.gnome.Nautilus.Tracker3.Miner.Files.service.in   | 4 ++--
 src/nautilus-tag-manager.c                               | 4 ++--
 6 files changed, 16 insertions(+), 15 deletions(-)

diff --git a/data/tracker/meson.build b/data/tracker/meson.build
index 5320af164c..2921ca925e 100644
--- a/data/tracker/meson.build
+++ b/data/tracker/meson.build
@@ -1,11 +1,12 @@
-# Files needed for running Tracker inside the Flatpak sandbox, for systems
-# which don't have a suitable version of Tracker in the host OS.
+# Files needed for running LocalSearch (historically known as Tracker) inside
+# the Flatpak sandbox, for systems which don't have a suitable version of
+# LocalSearch in the host OS.
 #
 # We must export the .service files from the sandbox so they work on the
-# session bus. This means the Tracker domain name must correspond with the
+# session bus. This means the LocalSearch domain name must correspond with the
 # application ID.
 
-domain_ontologies_dir = get_option('datadir') / 'tracker3' / 'domain-ontologies'
+domain_ontologies_dir = get_option('datadir') / 'localsearch3' / 'domain-ontologies'
 dbus_services_dir = get_option('datadir') / 'dbus-1' / 'services'
 
 tracker_domain_config = configuration_data()
diff --git a/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in b/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in
index eb7a87aa65..23ded45837 100644
--- a/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in
+++ b/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in
@@ -1,7 +1,7 @@
 [D-BUS Service]
 Name=@application_id@.Tracker3.Miner.Extract
-Exec=/app/libexec/tracker-extract-3 --domain-ontology @domain_rule@
+Exec=/app/libexec/localsearch-extractor-3 --domain-ontology @domain_rule@
 
-# Miner details needed for tracker-control
+# Miner details needed for localsearch-control
 Path=/org/freedesktop/Tracker3/Miner/Extract
 NameSuffix=Miner.Files
-- 
GitLab

From 25f445f4d7f38f501090cca88566ffd319d69463 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ant=C3=B3nio=20Fernandes?= <antoniof@gnome.org>
Date: Wed, 12 Jun 2024 11:24:24 +0100
Subject: [PATCH] tests: Use localsearch3 test-sandbox

Test sandbox program has been moved from the `TinySPARQL` (previously
`tracker`) project to `localsearch` (previously `tracker-miners`).

https://gitlab.gnome.org/GNOME/tinysparql/-/commit/067e855151cf47609bdeccadd9baa314016c4152
https://gitlab.gnome.org/GNOME/localsearch/-/commit/abc100fa6b259b9281231e6c8daf27723d62a3ed
---
 test/automated/displayless/meson.build | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/test/automated/displayless/meson.build b/test/automated/displayless/meson.build
index 9d50c5084e..13616a2608 100644
--- a/test/automated/displayless/meson.build
+++ b/test/automated/displayless/meson.build
@@ -1,6 +1,4 @@
-trackertestutils = dependency('tracker-testutils-3.0')
-
-tracker_sandbox = find_program(trackertestutils.get_variable('command'))
+tracker_sandbox = find_program('localsearch')
 
 tests = [
   ['test-directory', [
@@ -75,7 +73,7 @@ foreach t: tracker_tests
   test(
     t[0],
     tracker_sandbox,
-    args: ['--store-tmpdir', '--index-recursive-tmpdir', test_exe],
+    args: ['test-sandbox', '--store-tmpdir', '--index-recursive-tmpdir', test_exe],
     env: [
       test_env,
       'G_TEST_BUILDDIR=@0@'.format(meson.current_build_dir()),
-- 
GitLab

From 493858117e0e40052160b8e6df05bb4025e41d6c Mon Sep 17 00:00:00 2001
From: Bobby Rong <rjl931189261@126.com>
Date: Sat, 28 Sep 2024 15:29:01 +0800
Subject: [PATCH] nautilus-tag-manager: Drop tinysparql 2to3 migration

This reverts commit 7cf268ab73f71c09ed85f917c5c5f2703d71e400.

"tinysparql export --2to3" was gone in https://gitlab.gnome.org/GNOME/tinysparql/-/commit/e580569df5639bea171cf4211a31a2c5bdf809e9.
---
 src/nautilus-application.c |   1 -
 src/nautilus-tag-manager.c | 176 -------------------------------------
 src/nautilus-tag-manager.h |   2 -
 3 files changed, 179 deletions(-)

diff --git a/src/nautilus-application.c b/src/nautilus-application.c
index f13b4d7fe7..065eb053b4 100644
--- a/src/nautilus-application.c
+++ b/src/nautilus-application.c
@@ -1173,7 +1173,6 @@ nautilus_application_startup (GApplication *app)
     if (g_strcmp0 (g_getenv ("RUNNING_TESTS"), "TRUE") != 0)
     {
         maybe_migrate_gtk_filechooser_preferences ();
-        nautilus_tag_manager_maybe_migrate_tracker2_data (priv->tag_manager);
     }
 
     g_signal_connect (self, "shutdown", G_CALLBACK (on_application_shutdown), NULL);

